src/Compilers/CSharp/Portable/Symbols/ConstraintsHelper.cs

```diff
@@ -323,6 +323,32 @@ internal static class ConstraintsHelper
             diagnostics.Add(location, useSiteDiagnostics);
         }
 
+        internal static ImmutableArray<TypeParameterConstraintClause> MakeTypeParameterConstraints(
+            this SourceMethodSymbol containingSymbol,
```

Comment:
This calls another helper method that's also an extension method in this file. I'd prefer it to be a static helper or extension method because I don't want to accidentally take a dependency on the internals of the object, which this helper shouldn't need to know about.